app/helpers/dag_json_helper.rb

```diff
@@ -0,0 +1,119 @@
+module DagJsonHelper
+  DAG_NAME_HOST_FILTER = "host_filter".freeze
+  DAG_NAME_ALIGNMENT = "non_host_alignment".freeze
+  DAG_NAME_POST_PROCESS = "postprocess".freeze
+  DAG_NAME_EXPERIMENTAL = "experimental".freeze
+
+  def generate_dag_json(pipeline_run, dag_name)
```

Comment:
Consider removing this method and direct invoking `host_filtering_dag_attributes`, `alignment_dag_attributes`... directly.

You can create a private method in this module that contains this code:
```
def _render_dag_json(attribute_dict):
    sample = pipeline_run.sample
    dag = DagGenerator.new("app/lib/dags/#{dag_name}.json.erb",
                           sample.project_id,
                           sample.id,
                           sample.host_genome_name.downcase,
                           attribute_dict,
                           pipeline_run.parse_dag_vars)
    dag.render
```